PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Application/Services/Token/TokenService.cs

```diff
@@ -0,0 +1,48 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using PrivateHospitals.Application.Interfaces;
+using PrivateHospitals.Core.Models;
+
+namespace PrivateHospitals.Application.Services;
+
+public class TokenService : ITokenService
+{
+    private readonly IConfiguration _config;
```

Comment:
Use IOptions interface. Never inject IConfiguration directly to your services.
Move implementation of token service to Infrastructure layer because its infra concern. Though, `ITokenService` interface will remain on Application layer